### DISCUSSION TITLE
Setup pages doesn't inject correct the pathPrefix in Gatsby config

### DISCUSSION THREAD

**Discussion by torn4dom4n** [community/community#46521] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello, I use GitHub Pages action to config my Gatsby site. When I see the actions it shows that it injected success but actually my site has the wrong `pathPrefix` in the Gatsby config

![image](https://user-images.githubusercontent.com/27698189/217017901-5c5fab07-619d-44b7-9d30-bc61076ff9f0.png)

It shows `statics` instead of `nganguyenphotos`. This is [my repo](https://github.com/area44-stack/nganguyenphotos), [my lastest action log](https://github.com/area44-stack/nganguyenphotos/actions/runs/4105037421/jobs/7081349205), [my site](https://area44-stack.github.io/nganguyenphotos) and [my config site](https://github.com/area44-stack/nganguyenphotos/blob/main/gatsby-config.ts)

**Comment by yoannchaudet** [comment]
It looks to me like it is doing the right thing:

https://github.com/area44-stack/nganguyenphotos/actions/runs/4106429572/jobs/7084648894

![image](https://user-images.githubusercontent.com/14911070/217106421-34fa9a08-4e2e-4305-82ed-ec1da5984506.png)

Which you can confirm in the logs.

The theme you are using may not be reading this global property though. I see it has some dedicated `basePath` parameter you may want to pass yourself: https://www.gatsbyjs.com/plugins/@lekoarts/gatsby-theme-emilia/#theme-options.

**Reply by torn4dom4n** [reply]
Thanks for your support. I am not sure about why this happened. I have a [same repo](https://github.com/microlens/2022) which uses the same theme. My config has no `basePath` but it really works. The difference between the two repos is `siteURL`. Could you check this?

**Reply by torn4dom4n** [reply]
> It looks to me like it is doing the right thing:
> 
> https://github.com/area44-stack/nganguyenphotos/actions/runs/4106429572/jobs/7084648894
> 
> ![image](https://user-images.githubusercontent.com/14911070/217106421-34fa9a08-4e2e-4305-82ed-ec1da5984506.png)
> 
> Which you can confirm in the logs.
> 
> The theme you are using may not be reading this global property though. I see it has some dedicated `basePath` parameter you may want to pass yourself: https://www.gatsbyjs.com/plugins/@lekoarts/gatsby-theme-emilia/#theme-options.

I'll find new way. Thanks you.
